src/vdom.rs

```diff
@@ -313,13 +394,7 @@ impl<Ms, Mdl, ElC: View<Ms> + 'static, GMs: 'static> App<Ms, Mdl, ElC, GMs> {
         }
 
         // Our initial render. Can't initialize in new due to mailbox() requiring self.
```

Comment:
I think the first part is? We don't actually call view until here. The latter part might be stale though.

Reply:
Hrm, I stand corrected. This was always conditionally true since we call `update` if we pass `PassToRoutes` for `UrlHandling`. So it was out of date since the beginning, I guess.